apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V97__update_drgcode_add_claimquerycode_and_carrclmblgnpi.sql

```diff
@@ -0,0 +1,14 @@
+-- This script is to alter the clm_drg_code datatype size from 3 to 4 characters for
+-- the inpatient and snf claims tables, add the claim_query_code to the hha 
+-- and hospice claims tables, and add the carr_clm_blg_npi_num to the carrier claims table.
+
+-- Update DRG Code from 3 to 4 characters
+ALTER TABLE public.inpatient_claims ALTER COLUMN clm_drg_cd ${logic.alter-column-type} varchar(4);
+ALTER TABLE public.snf_claims ALTER COLUMN clm_drg_cd ${logic.alter-column-type} varchar(4);
+
+-- Add claim query code to the HHA Claims and Hospice Claims table.
+ALTER TABLE public.hha_claims ADD claim_query_code character(1);
+ALTER TABLE public.hospice_claims ADD claim_query_code character(1);
+
+-- Add carrier claim blg npi number to the Carrier Claims table.
+ALTER TABLE public.carrier_claims ADD carr_clm_blg_npi_num varchar(10);
```

Comment:
Making this comment here since I cannot comment on the spreadsheet:

Since these were added to the spreadsheet, I believe they will become columns on the generated entities, right? How are they being suppressed from the columns expected to be in the RIF?

Also, for DRG_CD, we need to have tests for both 3 and 4 character DRG_CD values in the RIFs

Reply:
Claim query code and carrier claim blg npi were not added to the spreadsheet.. The only change was to the  clm_drg_cd in inpatient and snf to change it from 3 to 4 characters.. Will add tests to the snf and inpatient entities.

Reply:
Added temporary code to not include those two new fields